src/modality-specific-files/physiological-recordings.md

```diff
@@ -51,10 +46,9 @@ the eyetracking data in a certain sampling frequency, and
 `sub-01_task-bart_recording-breathing_physio.tsv.gz` to contain respiratory
 measurements in a different sampling frequency.
 
-Physiological recordings (including eyetracking) SHOULD use the `_physio`
-suffix, and signals related to the stimulus SHOULD use `_stim` suffix.
+Physiological recordings (including eyetracking) SHOULD use the `_physio` suffix.
 
-The following tables specify metadata fields for the `*_<physio|stim>.json` file.
+The following table specifies metadata fields for the `*_physio.json` file.
```

Comment:
Actually because we've just split things into several tables, I think we should keep the `tables specify`, no?

Sorry about this.